symbology/apply-unique-values-with-alternate-symbols/README.metadata.json

```
@@ -0,0 +1,36 @@
+{
+    "category": "Symbology",
+    "description": "Apply a unique value with alternate symbols at different scales.",
+    "ignore": false,
+    "images": [
+        "ApplyUniqueValuesWithAlternateSymbols.gif"
+    ],
+    "keywords": [
+        "alternate symbols",
+        "multilayer symbol",
+        "scale based rendering",
+        "simple marker symbol",
+        "symbology",
+        "unique value",
+        "unique value renderer",
+        "MultilayerPointSymbol",
+        "SimpleMarkerSymbol",
+        "SymbolReferenceProperties",
+        "UniqueValue",
+        "UniqueValueRenderer"
+    ],
+    "redirect_from": [
+        ""
+    ],
```

this is one area with the metadata json that we run into issues with, so it's good you've come across this on your first sample! Basically, if there is no redirect, we must use the following notation i.e. an empty string within an array will not be accepted and the webpage will crash on loading.

```suggestion
    "redirect_from": "",
```

this has happened a couple of times, so when we updated our Github actions, we put in a check for this. If you have a look at the Readme check, and scroll down to the bottom, you'll see there a warning and a message as to why this check on the metadata json has failed.

https://github.com/Esri/arcgis-runtime-samples-java/pull/692/checks